### PR TITLE
Vodafone Gigabox router documentation

### PR DESCRIPTION
**Pull request by jdieter**
**Description:**
Documentation for Vodafone Gigabox router integration

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27872

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/vodafonegigabox_router.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Vodafone Gigabox"
+description: "Instructions on how to integrate Vodafone Gigabit router into Home Assistant."
+logo: vodafone.svg
+ha_category:
+  - Presence Detection
+ha_release: 0.100.3
```
Comment (klaasnicolaas) [inline source/_integrations/vodafonegigabox_router.markdown]:
```suggestion
ha_release: 0.101
```

File: source/_integrations/vodafonegigabox_router.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Configuration
+
+To use a Vodafone Gigabox router in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: vodafonegigabox_router
+    host: 192.168.1.1
```
Comment (klaasnicolaas) [inline source/_integrations/vodafonegigabox_router.markdown]:
```suggestion
    host: IP_ADDRESS
```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/vodafonegigabox_router.markdown (hunk 1 above)
Comment (jdieter) [reply source/_integrations/vodafonegigabox_router.markdown]: Fixed

File: source/_integrations/vodafonegigabox_router.markdown (hunk 2 above)
Comment (jdieter) [reply source/_integrations/vodafonegigabox_router.markdown]: Fixed

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
@jdieter I have trouble finding the parent PR, could you please link it? Thanks 👍

**Comment by jdieter** [comment]
Sorry, I haven't put it up yet.  I'm waiting for the tox tests to finish, and they're taking a lot longer than I expected.  It's getting late here, so I probably won't be able to get it posted until tomorrow.

**Comment by jdieter** [comment]
When I do, I'll link to it properly in the description

**Comment by jdieter** [comment]
Ok, I've added the PR for the parent in the description.  Apologies for the delay.

**Pull request closed without merging**